### PR TITLE
Migrated Bus Demo to 1.9.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

File: 1.9/README.md [hunk 1]
```diff
@@ -1,6 +1,266 @@
-Overview of demos in 1.9:
+#Iot Fast Data analytics
 
-* [Application Logs](applogs#fast-data-application-logs)
-* [Financial Transaction Processing](fintrans#fast-data-financial-transaction-processing)
-* [Sensor Analytics](sensoranalytics#fast-data-sensor-analytics)
-* [Tweeter](tweeter#tweeter)
+or how to run the SMACK stack on DC/OS with a stream of data.
```
Comment (unterstein) [inline 1.9/README.md]: `... or` ?

File: 1.9/configuration/akka-ingest.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "env": {
+    "CASSANDRA_HOST": "node-0.cassandra.mesos",
+    "CASSANDRA_PORT": "9042",
+    "KAFKA_HOST": "broker-0.kafka.mesos",
+    "KAFKA_PORT": "9092"
+  },
+  "upgradeStrategy": {
+    "minimumHealthCapacity": 0
+  }
+}
```
Comment (unterstein) [inline 1.9/configuration/akka-ingest.json]: do the empty line :P

File: 1.9/configuration/cassandra-schema.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  "run": {
+    "cmd": "/opt/bus-demo/import_data.sh node-0.cassandra.mesos",
+    "cpus": 0.1,
+    "mem": 256,
+    "disk": 0,
+    "docker": {
+      "image": "codecentric/bus-demo-schema:3.0.7"
+    }
+  }
+}
```
Comment (unterstein) [inline 1.9/configuration/cassandra-schema.json]: do it 🎉

File: 1.9/configuration/dashboard.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      "intervalSeconds": 60,
+      "timeoutSeconds": 20,
+      "maxConsecutiveFailures": 3,
+      "ignoreHttp1xx": false,
+      "port": 8000
+    }
+  ],
+  "cpus": 0.1,
+  "mem": 2048.0
+}
```
Comment (unterstein) [inline 1.9/configuration/dashboard.json]: :-/

**Comment by joerg84** [comment]
Will address in different PR.

**Pull request merged**